.travis.yml

```diff
@@ -9,6 +9,11 @@ services:
     - docker
 before_install:
     - docker pull gcr.io/google_containers/kubekins-job-builder:4
+# Install Go
```

Comment:
Travis has built-in support for installing Go versions, just like Python.

    go: 
        - 1.6.2